clients/go/cmd/zbctl/main_test.go

```diff
@@ -294,7 +318,9 @@ func (s *integrationTestSuite) runCommand(command []string, useHostAndPort bool,
 	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
 	defer cancel()
 
+	// set the request timeout to 1 more second to allow us to detect timeouts properly
```

Comment:
:wrench: Either this comment is not accurate or it is confusing. 1 more second than what?

Reply:
Yes, it's a leftover from rebasing. It shouldn't be there :see_no_evil: